mawaqit/android-tv-app
# feat: Add button to control sound volume

📝 **Summary**
---
**This PR for issue** #1220

**Description**
---
This PR introduces a new feature for adjusting the volume of the Quran player. It also includes several fixes and improvements to the player's UI and functionality.

Key changes:
1. Added a volume slider to the Quran player screen
2. Implemented volume control functionality in the Quran audio player notifier
3. Fixed directionality issues with player controls
4. Improved navigation controls and focus management
5. Added color customization for volume slider thumb

New features:
- Volume adjustment using a slider
- Volume toggle button to show/hide the volume slider
- Keyboard navigation for volume control

Bug fixes:
- Corrected icon directionality for RTL languages
- Improved focus traversal order for accessibility

UI improvements:
- Added visual feedback for focused controls
- Implemented responsive layout for player controls

**Tests**
---
🧪 **Use case 1: Volume Control (English - LTR)**
---
💬 **Description:**
1. Set app language to English
2. Open the Quran player screen
3. Tap the volume icon to reveal the volume slider
4. Adjust the volume using the slider
5. Verify that the audio volume changes accordingly
6. Test keyboard navigation for volume control

📷 **Screenshots or GIFs (if applicable):**

![en_quran_slider](https://github.com/user-attachments/assets/c1b41f43-790e-4abd-a923-8eaa19e793e6)

🧪 **Use case 2: Volume Control (Arabic - RTL)**
---
💬 **Description:**
1. Set app language to Arabic
2. Open the Quran player screen
3. Tap the volume icon to reveal the volume slider
4. Adjust the volume using the slider
5. Verify that the audio volume changes accordingly
6. Test keyboard navigation for volume control

📷 **Screenshots or GIFs (if applicable):**

![ar_slider_quran](https://github.com/user-attachments/assets/cb19f367-18a2-4967-822e-5020360c963d)

**Checklist:**
---
- [x] **Testing:** I have tested the changes and they work as expected in both English and Arabic interfaces.
